jenkins/scripts/files/run_integration_tests.sh

```diff
@@ -45,8 +45,10 @@ fi
 
 if [ "${REPO_NAME}" == "metal3-dev-env" ]
 then
-   METAL3REPO="${UPDATED_REPO}"
-   METAL3BRANCH="${UPDATED_BRANCH}"
+   export METAL3REPO="${UPDATED_REPO}"
+   export METAL3BRANCH="${UPDATED_BRANCH}"
+   export BAREMETAL_OPERATOR_LOCAL_IMAGE="https://github.com/metal3-io/baremetal-operator.git"
+   export CAPM3_LOCAL_IMAGE="https://github.com/metal3-io/cluster-api-provider-metal3.git"
```

Comment:
You should take the branch into account. if you are testing with v1alpha3 or v1alpha4, it is different branches in CAPM3 for example

Reply:
Fixed, thank you.